### PR TITLE
updated countdown timer placement to SubHeading Area

### PR DESCRIPTION
**Pull request by Klongrich**

### CODE REVIEW

File: src/components/EndTimer.tsx [hunk 1]
```diff
@@ -18,32 +18,16 @@ const EndTimer: React.FC<IEndTimer> = ({
...
-          expiryTimestamp === null ? Date.now() : expiryTimestamp
-        }
-      />
-      <Text fontSize={['12px', '14px']} fontWeight="light" mt="-20px">
-        {hardCapTimer} hour timer.
-      </Text>
-      <Text fontSize={['12px', '14px']} fontWeight="light">
-        Ends after {hardCapTimer} hours or {fromWei(hardcap)} ETH.
-      </Text>
-    </Box>
```
[the remark this answers is not in the document]
Comment (carlsbad-sunshine) [reply src/components/EndTimer.tsx:46]: Yes this is required

[the remark this answers is not in the document]
Comment (ethereal-nfts) [reply src/components/EndTimer.tsx]: @Klongrich Still waiting for the hardap timer display to be added back in

[the remark this answers is not in the document]
Comment (carlsbad-sunshine) [reply src/components/EndTimer.tsx]: @Klongrich Still waiting for the hardcap timer to be added back in

[the remark this answers is not in the document]
Comment (Klongrich) [reply src/components/EndTimer.tsx]: should be added

**Review by iamonuwa — CHANGES_REQUESTED** [review]
@Klongrich please remove the web3modal pop up on initial page load. Allow the user to call it first.

**Comment by Klongrich** [comment]
> @Klongrich please remove the web3modal pop up on initial page load. Allow the user to call it first.

Okay, Would you want to have that on a different PR? Just makes it more organized. (:

**Comment by Klongrich** [comment]
> @Klongrich please remove the web3modal pop up on initial page load. Allow the user to call it first.

Miss-clicked and closed PR on accident. Would you want to have that update on a new PR? I have some of the fixes on a different branch.

**Pull request closed without merging**